mm.c

```diff
@@ -464,16 +500,27 @@ static void test_GET_PTR_SUCC_FIELD()
 static void test_GET_SUCC()
 {
     printf("Test test_GET_SUCC.\n");
-    char **bp = malloc(3); // enough for hdr + pred field + succ field 
+    char **bp = malloc(3); // enough for hdr + pred field + succ field
     char **ptr_to_succ_field = bp + HDR_SIZE + PRED_FIELD_SIZE; // bp should point to the succ field after this line
 
     assert(GET_SUCC(bp) == (*ptr_to_succ_field));
     printf("Test passed.\n\n");
     free(bp);
 }
 
+static void test_GET_PRED()
+{
+    printf("Test test_GET_PRED.\n");
+    char **bp = malloc(2*WORD_SIZE); // enough for hdr + pred field
+    char **ptr_to_pred_field = bp + HDR_SIZE; // bp should point to the pred field after this line
```

Comment:
for the comment, shouldn't it be "ptr_to_pred_field should point to the pred field after this line"